Boundary Docs (#2907)
hashicorp/web-unified-docs: content/boundary/v1.0.x/content/docs/concepts/transparent-sessions.mdx

---
layout: docs
page_title: Transparent sessions in Boundary
description: >-

>   Connect to Boundary targets without entering resource IDs or port numbers — transparent sessions route traffic automatically once you're authenticated and have aliases configured.

Reviewer: ```suggestion
  Connect to Boundary targets without entering resource IDs or port numbers. Transparent sessions use aliases to route traffic automatically.
```

# START AUTO GENERATED METADATA, DO NOT EDIT
created_at: 2025-11-18T17:04:41-05:00
last_modified: 2026-01-26T13:07:13-05:00
# END AUTO GENERATED METADATA
---

> # Transparent sessions in Boundary

Reviewer: ```suggestion
# Transparent sessions
```
